src/main/scala/no/ndla/draftapi/service/WriteService.scala

```diff
@@ -92,14 +92,15 @@ trait WriteService {
                    oldNdlaCreatedDate: Option[Date],
                    oldNdlaUpdatedDate: Option[Date],
                    importId: Option[String]): Try[api.Article] = {
+      val withNotes = newArticle.copy(notes = Seq("Opprettet artikkel."))
```

Comment:
Her overskriver vi alle notes til en eventuelt ny artikkel. 
Tror vi heller burde lage en:
`val notes = "Opprettet artikkel." +: newArticle.notes`
Som kan puttes inn i `newNotes` funksjonen istedet for bare `newArticle.notes`

Reply:
Fiksa